README.md

```diff
@@ -1,61 +1,86 @@
-    │
-    ├── models             <- Trained and serialized models, model predictions, or model summaries
-    │
-    ├── notebooks          <- Jupyter notebooks. Naming convention is a number (for ordering),
-    │                         the creator's initials, and a short `-` delimited description, e.g.
-    │                         `1.0-jqp-initial-data-exploration`.
-    │
-    ├── references         <- Data dictionaries, manuals, and all other explanatory materials.
-    │
-    ├── reports            <- Generated analysis as HTML, PDF, LaTeX, etc.
-    │   └── figures        <- Generated graphics and figures to be used in reporting
-    │
-    ├── requirements.txt   <- The requirements file stating direct dependencies if a library
-    │                         is developed.
-    │
-    ├── setup.py           <- makes project pip installable (pip install -e .) so src can be imported
-    ├── src                <- Source code for use in this project.
-    │   ├── __init__.py    <- Makes src a Python module
-    │   │
-    │   ├── data           <- Scripts to download or generate data
-    │   │   └── make_dataset.py
-    │   │
-    │   ├── features       <- Scripts to turn raw data into features for modeling
-    │   │   └── build_features.py
-    │   │
-    │   ├── models         <- Scripts to train models and then use trained models to make
-    │   │   │                 predictions
-    │   │   ├── predict_model.py
-    │   │   └── train_model.py
-    │   │
-    │   └── visualization  <- Scripts to create exploratory and results oriented visualizations
-    │       └── visualize.py
-    │
-    ├── .thoth.yaml        <- Thoth's configuration file
-    ├── .aicoe-ci.yaml     <- AICoE CI configuration file (https://github.com/AICoE/aicoe-ci)
-    └── tox.ini            <- tox file with settings for running tox; see tox.readthedocs.io
-
-
---------
-
-<p><small>Project based on the <a target="_blank" href="https://drivendata.github.io/cookiecutter-data-science/">cookiecutter data science project template</a>. #cookiecutterdatascience</small></p>
+# Thoth Tutorial - manage your dependencies in Jupyter notebooks.
+
+This tutorial is used to show how to manage dependencies for Jupyter Notebooks to allow reproducibility and shareability. You will learn how to move from local to cloud development using [Operate First][1] environment and how you can enable contributions on your project.
+
+Once the tutorial is completed, you will be able to run your work on [Project Meteor][2].
+
+## Environment and Tools
+
+In this tutorial the following technologies are going to be used:
+
+- [jupyterlab-requirements][3] library. ([Project Thoth][4], JupyterLab extension for dependency management)
+- [JupyterHub][5] to launch images with Jupyter tooling.
+
+If you use the [Operate First][1] environment, the above requirements will be already setup for you selecting `elyra-image` from [JupyterHub][5].
+
+### Operate First Open Environment
```

Comment:
There are references to Operate First before this description of what it is.
Can we put the description earlier? Probably in the initial introduction?

Reply:
Totally make sense! Thanks @codificat !